### PR TITLE
JU-599 efs-submission-web - initial terraform and docker code changes

### PR DESCRIPTION
**Pull request by nghessen-ch**
the start of the main tf and docker code changes
need to remove eric, add healthcheck path + lb listener paths etc

### CODE REVIEW

**Comment by nghessen-ch** [comment]
need to still double check the healthcheck and lb listener paths and add them in

**Comment by nghessen-ch** [comment]
need to add readme template

**Comment by nghessen-ch** [comment]
readme updated, need to add link to new pipeline when it has been created.

File: README.md [hunk 1]
```diff
@@ -103,4 +103,32 @@ System properties for efs-submission-web are defined in `application.properties`
...
+
+The code present in this repository is used to define and deploy a dockerised container in AWS ECS.
+This is done by calling a [module](https://github.com/companieshouse/terraform-modules/tree/main/aws/ecs) from terraform-modules. Application specific attributes are injected and the service is then deployed using Terraform via the CICD platform 'Concourse'.
+
+
+Application specific attributes | Value                                | Description
+:---------|:-----------------------------------------------------------------------------|:-----------
+**ECS Cluster**        |filing-maintain                                      | ECS cluster (stack) the service belongs to
+**Load balancer**      |{env}-chs-chgovuk                                           | The load balancer that sits in front of the service
+**Concourse pipeline**     |[Pipeline link]({{LINK_TO_PIPELINE}}) <br> [Pipeline code](https://github.com/companieshouse/ci-pipelines/blob/master/pipelines/ssplatform/team-development/efs-submission-web)                                  | Concourse pipeline link in shared services
```
Comment (chillier-ch) [inline README.md:118]: need to update this first link

**Review by chillier-ch — APPROVED** [review]

**Pull request merged**